src/Sylius/Component/Core/Promotion/Reverser/OrderItemPromotionAdjustmentsReverserInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Promotion\Reverser;
+
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Promotion\Model\PromotionInterface;
+
+/**
+ * @author Gorka Laucirica <gorka.lauzirika@gmail.com>
+ */
+interface OrderItemPromotionAdjustmentsReverserInterface
+{
+    /**
+     * @param OrderInterface $order
+     * @param PromotionInterface $promotion
+     */
+    public function revert(OrderInterface $order, PromotionInterface $promotion);
```

Comment:
The interface is named `Reverser` but the method is named `revert`... Is that correct? I really don't know 😄